chapters/classes.tex

```diff
@@ -1183,10 +1183,10 @@ \section{Balanced Models}\label{balanced-models}
 end BaseProperties;
 \end{lstlisting}
 
-The use of connector here is a special design pattern.
+The model \lstinline!BaseProperties! together with its use in derived classes and as component relies on a special design pattern defined below.
 The variables \lstinline!p!, \lstinline!h!, \lstinline!Xi! are marked as input to get correct equation count.
 Since they are connectors they should neither be given binding equations in derived classes nor when using the model.
-The design pattern is to give textual equations for them (as below); using \lstinline!connect!-equations for these connectors would be possible (and would work) but is not part of the design.
+The design pattern is to give textual equations for them (as below); using \lstinline!connect!-equations for these connectors would be possible (and would work) but is not part of the design pattern.
```

Comment:
```suggestion
The design pattern in this case is to give textual equations for them (as below); using \lstinline!connect!-equations for these connectors would be possible (and would work) but is not part of the design pattern.
```

Reply:
You answer [here](https://github.com/modelica/ModelicaSpecification/issues/3242#issuecomment-1251128630) is absolutely clear to me. However, the revised text is still a bit ambiguous to me. I tried to make it stronger by adding "in this case".

What is not clear to me is how a user can tell apart cases where the design pattern is to actually use connections (as in Modelica.Blocks), and when instead one is expected to use textual equations. Can you comment on that=

Reply:
To me "in this case" would make it less clear. Will see if it can be improved in other ways.

Reply:
"The design pattern, which is used in this case, ..." seems to work for me.

Reply:
@casella ok with new variant?

Reply:
Pinging @casella again.

Reply:
@casella ?